nncf/tensorflow/quantization/quantizers.py

```diff
@@ -100,6 +113,26 @@ def mode(self) -> str:
         """
         raise NotImplementedError
 
+    @property
+    def signedness_to_force(self) -> Optional[bool]:
```

Comment:
Asymmetric quantizer does not have signedness_to_force attribute.

Reply:
Thanks for the comment! I understand your point of view.

*My suggestion:*

Since all quantizers have the instance of the `TFQuantizerSpec` class which describes the specification of the quantizer I suggest adding the `quantizer_spec` or `spec` property (I'm open to suggestions about the property name) to the `Quantizer` superclass.

After these changes, when we want to get, for example, the `num_bits` parameter, we use following
```
q.quantizer_spec.num_bits
```
or
```
q.spec.num_bits
```
I like the last option more. I remember, that main reason for these is providing one (common for `SymmetricQuantizer` and `AsymmetricQuantizer` classes) implementation of the `get_quantizer_config()`, `get_config()`, `from_config()` methods.

@alexsu52 What do you think about it?

Reply:
@andrey-churkin As I understand you want to create common implementation of the `get_quantizer_config()`, `get_config()`, `from_config()`. Could you make a base implementation of these methods in the base class and overdrive it in descendants if it's needed.